src/controller/controller.js

```diff
@@ -0,0 +1,59 @@
+const model = require("../models/user");
+
+//  Add new data
+exports.adddata = async (req, res) => {
+  try {
+    await model.create(({ userName, email, birthdate, age, password } = req.body));
+    res.status(201).send("Data Added sucessfully");
+  } catch (err) {
+    res.status(400).send(" Error in data Creation :" + err.message);
+  }
+};
+
+// View the user data
+
+exports.viewuser = async (req, res) => {
+  try {
+    const data = await model.findById(req.params.userId).lean();
+    res
+      .status(201)
+      .send(
+        "Hear is My Details : \n   Name : " +
+          data.name +
+          "\n Email :" +
+          data.email +
+          "\n Age : " +
+          data.age +
+          "\n Birthdate : " +
+          data.birthdate
+      );
+  } catch (err) {
+    res.send(err.message + "Fetching data is unsucessfull").status(400);
+  }
+};
+
+// Update the data using id
+
+exports.updatedata = async (req, res) => {
+  try {
```

Comment:
for the formatting i used the pretieer and it is well formated in vs code i think due to space isssue some lines are showing in git hub